kernelci.org/content/en/kernel-community/_index.md

```diff
@@ -14,4 +14,6 @@ This section of the documentation shares instructions to:
 * [enable specific tests](../maestro/pipeline/developer-documentation/#enabling-a-new-test)
 * [visualize the results](view-results)
 
+For suggestions of tests to enable after boot tests, see [device kselftests](device-tests).
```

Comment:
For suggestions of **device** tests...

Reply:
@padovan  did you mean for "devices" to be in bold?

I assumed you had it in bold in your comment to highlight the missing word in your suggestion...